### PR TITLE
Support for flutter_map ^6.0.0-dev.3

### PR DESCRIPTION
**Pull request by shrijanRegmi**
I wanted to try out flutter_map v6 in one of my projects, and I didn't want to wait, so I took the initiative to update the package myself. Please take a moment to review the pull request. There might be numerous potential improvements that you may want to consider.

Note: After this update, you may also need to make adjustments to the example project, as it might not work as expected.

### CODE REVIEW

**Comment by ibrierley** [comment]
Hi, thanks for this. It doesn't seem to work for me, and I get a lot of errors, especially around the plugin_api.dart requests.

I know this update came in linked to the beta, and not sure if that has differences here, but if you want to check it again with the latest release, and especially run the examples, I'll take another look. Thanks!

**Comment by shrijanRegmi** [comment]
Hey @ibrierley

I just got chance today to update the package to support new release of flutter_map: ^6.0.0. It's really late here in my time today, I'll update this PR tomorrow.

File: lib/src/drag_marker_widget.dart [hunk 1]
```diff
@@ -115,13 +117,7 @@ class DragMarkerWidgetState extends State<DragMarkerWidget> {
...
     final dragPos = _offsetToCrs(localPosition);
 
-    if (widget.mapState.isOutOfBounds(dragPos)) {
-      // cancels the dragging, needed when the app runs in a window and the
-      // cursor leaves the map while dragging. The on pan end event fails to
-      // fire when doing a quick movement
-      _end();
-      return;
-    }
+    // TODO: Please check if the bounds are correct or not before proceeding
```
Comment (josxha) [inline lib/src/drag_marker_widget.dart:120]: This pull request might introduce a bug on windowed devices.
The draggin might not stop when leaving the window while dragging a marker.

**Review by josxha — CHANGES_REQUESTED** [review]

**Comment by shrijanRegmi** [comment]
Closing this since a new pull request is sent for `flutter_map: ^6.0.0` #39

File: lib/src/drag_marker_widget.dart (hunk 1 above)
Comment (shrijanRegmi) [reply lib/src/drag_marker_widget.dart:120]: Thanks @josxha
Completely forgot about that in the rush.

Can you or @ibrierley add that check in the new PR #39 ?

**Pull request closed without merging**